File: vendaval.client/src/app/chat/message.ts
import { BaseModel } from "../shared/common/interfaces/BaseModel";
import { MessageMedia } from "./messagemedia";

export interface Message extends BaseModel{
  conversationId: number;
  senderId: number;
  receiverId: number;
  senderConnectionId: string;
  receiverConnectionId: string;
  content: string;
  media: MessageMedia[];
}

File: Vendaval.Domain/Entities/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Vendaval.Domain.ValueObjects;

namespace Vendaval.Domain.Entities
{
    public class Message : BaseModel
    {
        public int ConversationId { get; set; }
        public required string SenderConnectionId { get; set; }
        public required string ReceiverConnectionId { get; set; }
        public required string Content { get; set; }
        public List<MessageMedia>? Media { get; set; }
    }
}
